src/llama-model.cpp

```diff
@@ -836,6 +836,28 @@ void llama_model::load_hparams(llama_model_loader & ml) {
                     default: type = LLM_TYPE_UNKNOWN;
                 }
             } break;
+        case LLM_ARCH_MODERN_BERT:
+            {
+
+                hparams.swa_type = LLAMA_SWA_TYPE_SYMMETRIC;
+                hparams.set_swa_pattern(3);
```

Comment:
Is this intrinsic to the model architecture or something that should be pulled from config? In `granite-embedding-small-english-r2`'s `config.json`, I see `  "global_attn_every_n_layers": 3` which makes me think it should be configurable.